content/livesync/quickstart.textile

```diff
@@ -1,7 +1,240 @@
 ---
 title: Quickstart
-meta_description: "."
+meta_description: "A quickstart guide to learn the basics of integrating the Ably LiveSync product into your application."
 product: livesync
 languages:
   - javascript
 ---
+
+This quickstart will provide you with a set of instructions to get an example running locally on your machine. Following this, you'll cover each component of LiveSync found within the code of the example.
+
+h2(#example). The example
```

Comment:
I think this is a redundant title. Let's skip directly to 'Clone the repo' and bring it up one level.